server/server.go

```diff
@@ -53,6 +54,8 @@ type Server struct {
 	routinesPool                  *safe.Pool
 	leadership                    *cluster.Leadership
 	defaultForwardingRoundTripper http.RoundTripper
+	metricsRegistry               metrics.Registry
+	metricsEnabled                bool
```

Comment:
As discussed on Slack refactored now towards an `IsEnabled()` method on the `Registry`.